webui/react/src/components/ColumnPickerMenu.tsx

```diff
@@ -284,7 +280,7 @@ const ColumnPickerMenu: React.FC<ColumnMenuProps> = ({
                   ),
                   forceRender: true,
                   key: canonicalTab,
-                  label: locationLabelMap[canonicalTab],
+                  label: locationLabelMap[canonicalTab as keyof typeof locationLabelMap],
                 };
               })}
             />
```

Comment:
tbh i'm not really sure why we have a `tabs.length === 1` down two lines. are we planning on using that condition later?

Reply:
the Searches view is a case where we only display columns with location `EXPERIMENT`, so we don't need the `Pivot` component and can just display one `ColumnPickerTab` instead.

Reply:
do we ever do that anywhere?

Reply:
looks like there's only one reference right now
<img width="439" alt="image" src="https://github.com/determined-ai/determined/assets/5224116/8b320367-540a-4c9a-9c8d-136cc10cd616">